### PR TITLE
fix(crosshairs): reset crosshair angle and position on camera reset event

### PR DESCRIPTION
**Pull request by IbrahimCSAE**
### Context

Reset angle, position and MIP on all viewports linked to the crosshairs

### Changes & Results

https://github.com/cornerstonejs/cornerstone3D/assets/93064150/62f8bd4f-bf22-474d-bc6b-8535d0f9001a

### CODE REVIEW

File: packages/tools/src/eventDispatchers/cameraResetEventDispatcher.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+/**
+ * When the camera is reset, check what tools need to react to this.
+ *
+ * - First we get all tools which are active, passive or enabled on the element.
+ * - If any of these tools have a `onCameraReset` method, we call it.
+ *
+ * @param evt - The normalized camera reset event.
+ */
+const onCameraReset = function (evt: Types.EventTypes.CameraResetEvent) {
+  // @ts-ignore
```
Comment (sedghi) [inline packages/tools/src/eventDispatchers/cameraResetEventDispatcher.ts:16]: please fix this ignore

File: packages/core/src/RenderingEngine/RenderingEngine.ts [hunk 2]
```diff
@@ -626,7 +626,18 @@ class RenderingEngine implements IRenderingEngine {
...
+      const resetPan = true;
+      const resetZoom = true;
+      const resetToCenter = true;
+      const resetRotation = false;
+      const supressEvents = true;
+      vp.resetCamera(
+        resetPan,
+        resetZoom,
+        resetToCenter,
+        resetRotation,
```
Comment (sedghi) [inline packages/core/src/RenderingEngine/RenderingEngine.ts:638]: i don't see we have resetRotation, did you mean storeAsInitial camera false?

Comment (IbrahimCSAE) [reply packages/core/src/RenderingEngine/RenderingEngine.ts:638]: image

Comment (IbrahimCSAE) [reply packages/core/src/RenderingEngine/RenderingEngine.ts:638]: image

Comment (IbrahimCSAE) [reply packages/core/src/RenderingEngine/RenderingEngine.ts:638]: resetRotation goes through VolumeViewport, the first 3 go through the super

Comment (sedghi) [reply packages/core/src/RenderingEngine/RenderingEngine.ts:638]: but then the part you are touching is in rendering engine, and it is vtkJSDrivenViewports which is both Istack and I volume

**Pull request merged**